master/security/calico-network-policy.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Get started with calico network policy
+---
+
+### Big Picture
+Enforce the network traffic that is allowed or denied using rules in Calico network policy. 
```

Comment:
This sentence feels a bit awkward to me. Perhaps this is all it needs?
```suggestion
Enforce which network traffic is allowed or denied using rules in Calico network policy. 
```

Reply:
That is good catch. This doc has been through multiple reviewers and edits so I appreciate that you are doing a final look.

Reply:
Fixed.